src/pages/contribute/index.tsx

```diff
@@ -0,0 +1,195 @@
+import Head from 'next/head';
+import Link from 'next/link';
+import {
+  Button,
+  Flex,
+  View,
+  Heading,
+  Text,
+  Link,
+  Card,
+  Icon
+} from '@aws-amplify/ui-react';
+import { FiExternalLink } from 'react-icons/fi';
+import Layout from './_components/Layout';
+import HowItWorks from './_components/HowItWorks';
+import QuickstartResources from './_components/QuickstartResources';
+
+// const { Octokit } = require("@octokit/rest");
+
+export async function getStaticProps() {
+  // const octokit = new Octokit({
+  //   auth: process.env.TOKEN,
+  // });
+
+  // let { data: JSissues } = await octokit.rest.issues.listForRepo({
+  //   owner: "aws-amplify",
+  //   repo: ["amplify-js"],
+  //   state: "open",
+  //   labels: ["good first issue"],
+  //   per_page: 6,
+  // });
+
+  // let { data: CLIissues } = await octokit.rest.issues.listForRepo({
+  //   owner: "aws-amplify",
+  //   repo: ["amplify-cli"],
+  //   state: "open",
+  //   labels: ["good first issue"],
+  //   per_page: 6,
+  // });
+
+  const JSissues = {};
+  const CLIissues = {};
+
+  return {
+    props: { JSissues, CLIissues },
+    revalidate: 60 * 60
+  };
+}
+
+export default function Index({ JSissues, CLIissues }) {
+  const meta = {
+    title: 'AWS Amplify Contributor Program',
+    description:
+      'The Amplify Contributor Program is an open invitation for you to participate in the Amplify open source development journey. Get involved with AWS Amplify by making open source contributions to the Amplify project!'
+  };
+  const url = 'https://docs.amplify.aws/contribute';
+  const creator = '@AWSAmplify';
+  return (
+    <>
+      <Head>
+        <title>AWS Amplify Contributor Program</title>
+
+        <meta name="HandheldFriendly" content="True" />
+        <meta name="MobileOptimized" content="320" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+        <meta name="description" content={meta.description} />
+
+        <meta property="og:title" content={meta.title} key="title" />
+        <meta property="og:site_name" content="Amplify Contributor Program" />
+        <meta property="og:description" content={meta.description} />
+        <meta property="og:image:type" content="image/png" />
+        <meta property="og:url" content={url} />
+        <meta property="og:locale" content="en-us" />
+
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:site" content={creator} />
+        <meta name="twitter:creator" content={creator} />
+        <meta name="twitter:title" content={meta.title} />
+        <meta name="twitter:description" content={meta.description} />
+        {/* <meta
+          property="twitter:image"
+          content="https://contributetoamplify.com/contributor-bash-og-image.png"
```

Comment:
@todo add proper/correct opengraph image

Reply:
@josefaidt this gtg?

Reply:
yep, it's been updated

```
<meta
          property="twitter:image"
          content="/images/contribute/contributor-bash-og-image.png"
        />
```